api/lib/utils/schedulerUtils.js

```diff
@@ -1,11 +1,17 @@
 'use strict';
-module.exports = function(settings, tlsOptions) {
```

Comment:
Why do we pass in the entire settings when all it needs is the scheduler portion?